backend/Dockerfile

```diff
@@ -1,4 +1,19 @@
-FROM django:latest
+FROM python:3.4-slim
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    gcc \
+    gettext \
+    mysql-client \
+    libmysqlclient-dev && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+## We COULD install Django from here, but it will be overriden
+## by the requirements-file ...
+# ENV DJANGO_VERSION 1.11.3
+# RUN pip install mysqlclient django=="$DJANGO_VERSION"
+
+RUN pip install mysqlclient
```

Comment:
I don't think we need this at all. If Django does not already this dependency, using a mysql client can be done from the db container.

Can you test if the stack works without it?